site/source/tutorials/007_theming/index.md

```diff
@@ -100,37 +112,36 @@ We can now use these variables in `worker.m.css` to create our fully themed `wor
 
-{% instruction 'Lets create `workerForm.m.css` and include it in our theme.ts' %}
+{% instruction 'Let\'s create `workerForm.m.css` and include it in theme.ts' %}
 
 {% include_codefile 'demo/finished/biz-e-corp/src/themes/dojo/workerForm.m.css' lang:css %}
 
 {% include_codefile 'demo/finished/biz-e-corp/src/themes/dojo/theme.ts' lines:3,5,9,11,13,16 %}
 
-This should be familiar to you by now as we covered theming `worker` in the last section. Now to theme the Dojo 2 `TextInput` that `WorkerForm` uses, we will need to create a `dojoTextInput.m.css` file and export it from `theme.ts` using a special theme key which is prefixed with `dojo-`. This prefix helps to ensure that our own widget theme files do not clash with dojo specific widgets and makes it easy to recognise when we are targeting one of the other.
+This should be familiar from theming the `worker` in the previous section. To theme the Dojo 2 `TextInput` within our `WorkerForm`, we need to create `dojoTextInput.m.css` and export it from `theme.ts` using a theme key prefixed with `dojo-`. The dojo prefix helps ensure that the dojo widget theme keys don't clash with application widget theme keys.
 
-{% instruction 'Create `dojoTextInput.m.css` and emport it from `theme.ts`' %}
+{% instruction 'Create `dojoTextInput.m.css` and export it from `theme.ts`' %}
 
 {% include_codefile 'demo/finished/biz-e-corp/src/themes/dojo/dojoTextInput.m.css' lang:css %}
 
 {% include_codefile 'demo/finished/biz-e-corp/src/themes/dojo/theme.ts' lines:3,5,6,9,11,13,14,16 %}
 
-In this example we are introducing another powerful part of the Dojo 2 theming system, `composes`. Composes comes from the `css-module` specification and allows you to take styles from one class selector and apply them to another. In this case we are saying that `root` of a `TextInput` (the label text in this case), should look the same as the `nameLabel` class in our `WidgetForm`. This approach can be very useful when creating multiple themes from a baseTheme and avoids re-writing / re-specifying styles in multiple places.
+In this example we are introducing another powerful part of the Dojo 2 theming system, `composes`. Composes originates in the `css-module` specification, allowing you to apply styles from one class selector to another. Here we are specifying that `root` of a `TextInput` (the label text in this case), should appear the same as the `nameLabel` class in our `WidgetForm`. This approach can be very useful when creating multiple themes from a `baseTheme` and avoids repetitive redefinition of style rules.
 
-If you open up your web browser now you should see that the Text Inputs at the top of the form have been styled.
+In your web browser you will see the `TextInputs` at the top of the form have been styled.
```

Comment:
should probably be `TextInput` widgets (since TextInputs is not a valid class name